### ISSUE TITLE
Adjust learning rate dynamically without resetting the network

### ISSUE THREAD

**Issue by botonchou** [tensorflow/playground#4]
I tried to lower the learning rate after several hundred iterations. But it seems like it'll clear all previous results and re-initialize all the weights.

Is it possible to adjust the learning rate online?
(I can imagine this requires some works on the URI.)

Thanks

**Comment by dsmilkov** [comment]
Thanks! Fixed in #6 . You will now be able to change the learning rate without resetting the model.

**Comment by emmanuelrouxfr** [comment]
Hello there ! 
We did some work in this same direction, and you will now be able to change the learning rate, the activation function, the regularization terms, etc ...  without resetting the model ! And this for each neuron individually (or mutually with a multiple-neuron selection) !

https://coolearning.github.io/playground

See you there :)
